notebooks/Medical Image Generation with Generative AI.ipynb

```diff
@@ -215,9 +218,10 @@
     "data = {\n",
     "    \"action\": \"generate\",\n",
     "    \"specs\": {\n",
-    "        \"num_output_samples\": 2,\n",
+    "        \"num_output_samples\": 1,\n",
```

Comment:
Can we have inline comments about the specs used for Gen2D? Thanks!

Reply:
Added the comment for each spec of GenAI. Thanks